### PR TITLE
Profile page made dynamic

### PR DESCRIPTION
**Pull request by Riya123-code**

### CODE REVIEW

File: assets/vendor/js/dashboard.js [hunk 1]
```diff
@@ -1741,7 +1741,7 @@ function loadSettingsCookies(closePop_up){
 
 function takeThemTo(whereTo){
   if(whereTo === "render"){
-    url = "view_render.php";
+    url = "ui_dashboard.php";
```
Comment (NtemKenyor) [inline assets/vendor/js/dashboard.js]: why was this changed from "view_render.php" to "ui_dashboard.php"

File: assets/vendor/js/profile_page.js [hunk 2]
```diff
@@ -19,9 +20,59 @@ function saveBio() {
...
-  var bio = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>";
-  var x = document.getElementById("def_block");
-  x.innerHTML = bio;
+  console.log("hii");
+  // var bio = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>";
+  // var x = document.getElementById("def_block");
+  // x.innerHTML = bio;
   var y = document.getElementById("textarea");
   y.innerHTML = bio;
+  $.post("../../backend/profile_page.php", {
```
Comment (NtemKenyor) [inline assets/vendor/js/profile_page.js]: sending a post request but profile_page.php was built for a GET request. For more improved code, U can make the correction in profile_page.php
change $_GET to $_POST

File: assets/vendor/js/profile_page.js [hunk 3]
```diff
@@ -19,9 +20,59 @@ function saveBio() {
...
+    
+  },
+  function(data, status){
```
Comment (NtemKenyor) [inline assets/vendor/js/profile_page.js:32]: Did not check for proper connection(e.g status == 200 etc).

File: pages/profilepage.html [hunk 4]
```diff
@@ -16,33 +16,48 @@
     rel="stylesheet"
     href="https://unpkg.com/swiper@7/swiper-bundle.min.css"
     />
+    <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
```
Comment (NtemKenyor) [inline pages/profilepage.html]: We already have JQuery in this project. You can import it locally as seen in other files. No need connecting to an unsecured endpoint as to fetch it.

File: pages/profilepage.html [hunk 5]
```diff
@@ -16,33 +16,48 @@
...
+
     <script type="text/javascript" defer src="../assets/vendor/js/profile_page.js"></script>
     <title>Profile-Page</title>
 </head>
+<script>
+function profile(){
+$.post("../../backend/profile.php", {
+				id:$_SESSION['userid'],
```
Comment (NtemKenyor) [inline pages/profilepage.html]: using PHP "$_SESSION" on this html file may not be very proper. Do you actually need it? I can't also find the file called "backend/profile.php"

File: pages/profilepage.html [hunk 6]
```diff
@@ -16,33 +16,48 @@
...
                     <div class="profile-pic-container">
                         <img src="../assets/img/anonymous_image.png" alt="Anonymous Profile Pic" class="profile-pic">
 
                     </div>
-                    <p class="username">Jill Anderson</p>
-                    <p class="user-title">UI Designer</p>
-                    <p class="user-description"><i class="fa fa-quote-left quote quote-left" aria-hidden="true"></i>Spend my days designing beautiful websites and applications with a focus on User Experience. Execute on the full spectrum, from ideation to mock-ups to coding to user testing. Live by the words: you can't use up creativity.<i class="fa fa-quote-right quote quote-right" aria-hidden="true"></i></p>
+                    <p class="section-content" id="nameid"></p>
+                    <p class="user-title" id="designation">UI Designer</p>
+                    <p class="user-description" id="intro"><i class="fa fa-quote-left quote quote-left" aria-hidden="true"></i><i class="fa fa-quote-right quote quote-right" aria-hidden="true"></i></p>
```
Comment (NtemKenyor) [inline pages/profilepage.html]: from your usage of "innerHTML" in the profile_page.js, You would be overriding the quotation marks.
Something like this can be the simplest solution.

`<p> <i> <span id="intro"> </span> </i></p>`

**Review by NtemKenyor — CHANGES_REQUESTED** [review]
Thank you for making your PR. Please check the errors pointed out.

File: pages/profilepage.html (hunk 6 above)
Comment (Riya123-code) [reply pages/profilepage.html:54]: sir can you please give more explanation on this as i did not get it.

**Comment by Riya123-code** [comment]
Hello sir,
Are there any changes i am supposed to make apart from above mentioned changes?(above mentioned changes are already done in last two commits) as this commit is  not getting merged.

**Comment by NtemKenyor** [comment]
Hello. Hope you're ok?

the truth about solving a problem is that another problem or need always
comes up.😊
Come to think of it: Now that a user can get content from the database,
How can a user input or edit the content in his profile? A form or a prompt
may be needed to edit or change the content of the users profile in the
frontend section and an endpoint in the backend repo...

But for now, We are good. 👍❤️😁

**Comment by Riya123-code** [comment]
Ok sir

**Review by NtemKenyor — APPROVED** [review]
Thank you for your contribution to Open Source development here at "Kwickerhub". We are not just happy but we excited to have someone as skilled as you to be part of our community.

Thank you and congratulations.
